common/lib/xmodule/xmodule/js/src/capa/display.js

```diff
@@ -1157,58 +1154,64 @@
          */
         Problem.prototype.disableAllButtonsWhileRunning = function(operationCallback, isFromCheckOperation) {
             var that = this;
-            this.enableAllButtons(false, isFromCheckOperation);
+            var allButtons = [this.resetButton, this.saveButton, this.showButton, this.hintButton, this.submitButton];
+            var initiallyEnabledButtons = allButtons.filter(function(button) {
+                return !button.attr('disabled');
+            });
+            this.enableButtons(initiallyEnabledButtons, false, isFromCheckOperation);
```

Comment:
So enableButtons is no longer defined on the class instance itself, just the prototype (since you deleted lines 39-41 above). But calling it here works because this is already a prototype method?